examples/postgresql/main.tf

```diff
@@ -43,13 +43,13 @@ module "aurora" {
     static = {
       identifier     = "static-custom-endpt"
       type           = "ANY"
-      static_members = ["static-member-1"]
+      static-members = ["static-member-1"]
```

Comment:
These changes are not correct. See code on line 215 - https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/pull/327/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR215